### DISCUSSION TITLE
Gotek not loading disk 2 in most Amiga games/softwares

### DISCUSSION THREAD

**Discussion by mister2cc** [keirf/flashfloppy#458] [Q&A]
Hi,
I recently bought a gotek and flashed it through usb ttl/serial adapter to the latest flashfloppy firmware.
The gotek is detected and all single game disks appear to work, but every game on more than 1 disk won't work because the loading process hangs as soon as I insert disk 2.
Any ideas?
This happens in all of my 3 amigas (600, 600, 1200) and both using the gotek as internal or external device.
Thanks in advance

**Comment by keirf** [comment]
Maybe you have a jumper at JC? Remove it if so.

**Reply by mister2cc** [reply]
Hi man, thanks for the advice. With the jumper completely removed it doesn't work, but on S0 it is working like never before! It loads everything  :) thanks!
